### ISSUE TITLE
Feature request - adding action icons that float over header image

### ISSUE THREAD

**Issue by Zordid** [ManuelPeinado/FadingActionBar#9]
Just like in the Google Music app, it would be awesome to easily attach "floating" action buttons on top of a list, hovering over the header image but detached from it when you slide your list, so that it basically seems to be linked to the topmost list item.
See red rectangle around the button in my screenshot.

![image](https://f.cloud.github.com/assets/4798822/722397/264e5742-e000-11e2-9657-643855ed7f06.png)

If that could also be accomplished by the FadingActionBarHelper - wow! ;-)

**Comment by ManuelPeinado** [comment]
Hi, thanks for the suggestion! However, I can't see that button when I run Play Music in my device, so it's hard for me to get a clear notion of how it works. Can you post a couple more snapshots illustrating the position of the button as the user scrolls through the content?

**Comment by Zordid** [comment]
I'll try to catch some more shots later. Basically in my screenshot above, the button is attached to the header "Meine Musik" and floats over the background image (the gray one with the headset face). It also moves underneath the action bar of course!

**Comment by ManuelPeinado** [comment]
I see, thanks for the explanation. I'll try to come up with a way to do
this, stay tuned.

**Comment by ManuelPeinado** [comment]
Hi.

I've implemented your suggestion, and it looks good! I've introduced the notion of "header overlay", which is a layout that is placed on top of the header, and which doesn't scroll with parallax.

Here's the apk, in case you want see the effect for yourself:

https://docs.google.com/file/d/0B42LZc5R_d4lV3lJa0JXdDN6aU0/edit?usp=sharing

To see how a header overlay is added to an activity, check out the "HeaderOverlayActivity" class in the "dev" branch:

https://github.com/ManuelPeinado/FadingActionBar/blob/dev/sample/src/com/manuelpeinado/fadingactionbar/demo/HeaderOverlayActivity.java

I will release this code as part of the next version of the library, probably next week.
